### DISCUSSION TITLE
High strength magnet steels

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2890] [Ideas]
In GitLab by @mkovari 

This paper proposes a new steel for the magnet conductor jackets and magnet casings: N50 - 

[Mechanical Properties Evaluation of ReBCO CICC Jacket Based on Super-Austenitic Stainless Steel for CFETR High-field Magnet](https://www.sciencedirect.com/science/article/pii/S2352179122002253)

> Nitronic 50 (N50) super-austenitic stainless steel material has great optimization potential for the development of jacket. ... The N50 circle-in-square jackets ... showed a yield strength greater than 1600 MPa and a fracture toughness K<sub>IC</sub> better than 110 MPa·m<sup>1/2</sup> after cold work at 4.2 K.

>...it is also necessary to evaluate the fatigue performance and welding performance in the future.

It might be sensible to use these parameters for DEMO and STP runs.

I am not sure if there is any way to infer fatigue properties from the data in the paper above, but see [The research on high-strength CICC jackets with YS> 1500 MPa at 4.2 K for future fusion applications](https://www.sciencedirect.com/science/article/pii/S2352179123001138), which does give the fatigue crack growth rate and other properties for N50H.

@ajpearcey @kj5248

**Comment by jmorris-uk** [comment]
I see no reason for us to not allow for different structural steels in the code.

Although we rarely change the temperature, it should be noted that we allow the user to change the operating temperature of the magnets. We should probably add a warning about stress properties being defined at a different temperature to the magnet operating temperature.

I'm not sure how prevalent yield strength vs. temperature data is or whether it is mainly room temp and 4.2/4.5K.

**Comment by mkovari** [comment]
There is some info on Japanese steel development in [Design study of superconducting coil system for JA DEMO](https://www.sciencedirect.com/science/article/pii/S0920379624001984#sec0007).

**Comment by mkovari** [comment]
See also [Nitronic 50 Steel as a Structural Material for EU-DEMO's Toroidal Field Coils](https://scholar.google.co.uk/scholar_url?url=https://infoscience.epfl.ch/bitstreams/4a2caa2f-e6df-4a41-859d-63da62e7ef28/download&hl=en&sa=X&d=3329002475033030381&ei=_t1oZ-XaNLiC6rQP-7vSmAc&scisig=AFWwaeae8eZcJdbTcfPSmFs9vAOY&oi=scholaralrt&html=&pos=0&folt=cit&fols=)
